### ISSUE TITLE
Load topological relationships from configuration files.

### ISSUE THREAD

**Issue by PatrickNa** [b-it-bots/topological_map#1]
**Is your feature request related to a problem? Please describe.**
The existing code has been written for the scope of the "Where is this?" scenario (GermanOpen 2019). At this point a working solution was the most important point and not much effort has been put into making it flexible considering various programming patterns. 
Therefore the topology is partially hard coded in the executing script. 

**Describe the solution you'd like**
The description of location in relation to each other should be possible defined in a config and loaded at execution time. This makes it flexible and we can apply this code to many scenarios/ environments.

**Comment by argenos** [comment]
@PatrickNa for which task is this?

**Comment by PatrickNa** [comment]
I actually added all the labels in the other repo... It is *mainly* "where is this".
I hope this issue will not be forgotten in this repo...

**Comment by argenos** [comment]
The implementation has to be done here, so it doesn't make sense to have it in `mas_domestic`. To "not forget issues" we should be using projects.

**Comment by argenos** [comment]
I opened a general issue where you can add the issues from other repos if that helps to manage things, but the details should go to the respective repos.

**Comment by PatrickNa** [comment]
Fair enough. I'm glad we have a solution for this particular *problem*, now.
